frontend/src/components/viewers/ROCCurve.tsx

```diff
@@ -105,9 +105,9 @@ class ROCCurve extends Viewer<ROCCurveProps, ROCCurveState> {
   }
 
   public render(): JSX.Element {
-
-    const width = this.props.maxDimension || 600;
+    const width = this.props.maxDimension || 800;
     const height = width * 0.65;
+    const isSmall = width < 600;
```

Comment:
We already pass the `maxDimension` as a prop, my goal so far was to let viewers decide how they want to render given that one number.

Reply:
I'm fine with you leaving it, I just thought it was a bit strange to have behavior controlled by size like that, rather than directly through some flag.

Reply:
Viewers are treated differently from other components though, I planned for them to behave more like black boxes. They have enough information about what they can display given a certain size, but their parents don't have that insight.